modules/command/common/dataset-pruning-command.js

```diff
@@ -43,7 +45,7 @@ class DatasetPruningCommand extends Command {
 
         forked.on('message', async (response) => {
             if (response.error) {
-                this.logger.error(`Error while pruning datasets. Error message: ${response.error.message}`);
+                this.logger.error(`Error while pruning datasets. Error message: ${response.error.message}. Pruning command will be executed again in ${constants.DATASET_PRUNING_COMMAND_TIME_MILLS / (1000 * 60 * 60 * 24)} hours`);
```

Comment:
I guess we don't need to multiply with 24, it that case we get days after the command will be executed, and not hours as expected

Reply:
good catch, thanks :-)